cocos/2d/CCScene.cpp

```diff
@@ -167,7 +172,7 @@ void Scene::render(Renderer* renderer)
         //clear background with max depth
         camera->clearBackground(1.0);
         //visit the scene
-        visit(renderer, transform, 0);
+        visit(renderer, transform, Node::FLAGS_TRANSFORM_DIRTY);
```

Comment:
- hitTest logic is like a 3D version "Node.contertToNodeSpce + Rect.containPoint", for convenience add it to Node, or any ideas?
- If this flag don't set to Node::FLAGS_TRANSFORM_DIRTY, the node's transform matrix can't update correctly. Like this:
* if let playerDlg, which in Scene3DTest.cpp, do rotage action(code below), the result is:
- `_playerDlg->runAction(RotateBy::create(5000, Vec3(360000, 360000, 0)));`
![cpp_tests](https://cloud.githubusercontent.com/assets/488183/8124136/196bc3c0-1109-11e5-9a0c-319222d126ea.png)